packages/@aws-cdk/aws-rds/lib/engine.ts

```diff
@@ -0,0 +1,267 @@
+import * as secretsmanager from '@aws-cdk/aws-secretsmanager';
+import { compare } from './private/version';
+import { ParameterGroupFamily } from './props';
+
+/**
+ * Base properties for all types of engines
+ * (cluster and instances).
+ */
+export interface BaseEngineProps {
```

Comment:
Can this be not exported?

Reply:
No, it has to be exported (otherwise JSII fails).